hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/accessor/entities/TraceEntityAccessor.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.trace.accessor.entities;
+
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.StructuredTrace;
+
+public interface TraceEntityAccessor {
```

Comment:
the previously parameterized version was always used with the same parameters, so removed them.